apps/amiapp_flutter/android/app/google-services.json

```diff
@@ -67,10 +103,10 @@
               "client_type": 3
             },
             {
-              "client_id": "1001564516023-5n3nd9b1d58s4pb776qpld55vuvrk45u.apps.googleusercontent.com",
+              "client_id": "1001564516023-4ec1vok624lq3sg3lu8mc8gbtn07bkbn.apps.googleusercontent.com",
               "client_type": 2,
               "ios_info": {
-                "bundle_id": "io.customer.ios-sample.cocoapods-fcm"
+                "bundle_id": "io.customer.rn-sample.fcm"
```

Comment:
Just checking, was this changed on purpose ?

Reply:
Yes, it was changed on purpose. By running the command, `flutterfire configure` it made the file change for me. My guess is the file downloaded the latest version of that file for us. We could retrieve this same file by downloading it manually from Firebase console.